concreep-refilled/control.lua

```diff
@@ -1,17 +1,17 @@
-MINIMUM_ROBOTS = 30
-MINIMUM_ITEM_COUNT = 200
+DEFAULT_MINIMUM_ROBOTS = 30
+DEFAULT_MINIMUM_ITEM_COUNT = 200
 
-NO_WORK_UPDATES_BEFORE_REINIT = 60
-SURFACES_PER_REINIT_PASS = 1
-ROBOPORTS_PER_REINIT_PASS = 1
+DEFAULT_NO_WORK_UPDATES_BEFORE_REINIT = 60
+DEFAULT_SURFACES_PER_REINIT_PASS = 1
+DEFAULT_ROBOPORTS_PER_REINIT_PASS = 1
 
 -- how many ups between creeper updates
-RUN_EVERY_N_UPDATES = 60
+DEFAULT_RUN_EVERY_N_UPDATES  = 60
 -- how many creepers to creep per update run
-CREEPERS_PER_UPDATE = 5
-INITIAL_RADIUS = 1
+DEFAULT_CREEPERS_PER_UPDATE = 5
+DEFAULT_INITIAL_RADIUS = 1
 
-DEBUG_PREFIX = "Concreep: "
+DEFAULT_DEBUG_PREFIX = "Concreep: "

```

Comment:
this one can stay in this file and doesn't need DEFAULT_ prefix, it's just a regular constant

Reply:
Done